js/runtime.ts

```diff
@@ -140,9 +141,13 @@ export function makeDefine(fileName: string): AmdDefine {
         return deno;
       } else {
         const resolved = resolveModuleName(dep, fileName);
-        const depModule = FileModule.load(resolved);
-        depModule.compileAndRun();
-        return depModule.exports;
+        util.assert(resolved != null);
```

Comment:
These null assert checks are redundant when using the bang operator. Suggest removing the assert.

Reply:
@ry just to make it clear, the not null assertion operator (`!`) in TypeScript only says "I know what I am doing" and does not actually change the runtime behaviour in any sense.  Given that, should we still remove it, or is it more valuable to actually put a comment in the assert to make the potential throw more logical.

As a side note, the TypeScript team recently closed the issue that would allow `assert` like calls to affect CFA analysis (Microsoft/TypeScript#8655) in lieu of Microsoft/TypeScript#10421 which _might_ allow authoring of some sort of function(s) that would allow this pattern to affect CFA and therefore the type past this point in the block.

Reply:
@kitsonk Oh sorry - I thought it threw an exception. In that case ignore my comments and keep the asserts.